src/parquet/encodings/plain-encoding.h

```diff
@@ -52,6 +52,19 @@ class PlainDecoder : public Decoder {
     return GetValues(buffer, max_values, sizeof(int64_t));
   }
 
+  virtual int GetInt96(Int96* buffer, int max_values) {
+    max_values = std::min(max_values, num_values_);
+    int size = 3*sizeof(uint32_t);//Struct Int96 could be padded
```

Comment:
Yes, but it would be convenient if `sizeof(Int96)` were always 12. I don't know the performance implications for unaligned memory access. This would be especially useful a template context, but it's easy to specialize the templates for `INT96`, too.